scripts/runChain.js

```diff
@@ -1,10 +1,19 @@
 const Blockchain = require("../classes/blockchain");
 const Block = require("../classes/block");
+const Transaction = require("../classes/transaction");
 
-const blockchain = new Blockchain();
+// const blockchain = new Blockchain();
 
-blockchain.addBlock(new Block(1, { amount: 4 }));
-blockchain.addBlock(new Block(2, { amount: 8 }));
+// blockchain.addBlock(new Block(1, { amount: 4 }));
+// blockchain.addBlock(new Block(2, { amount: 8 }));
 
-console.log("Blockchain is valid:", blockchain.isChainValid());
-console.log(JSON.stringify(blockchain, null, 2));
+// console.log("Blockchain is valid:", blockchain.isChainValid());
+// console.log(JSON.stringify(blockchain, null, 2));
+
+const transaction = new Transaction("123", "123", "1", 1, "123", 1, 1);
+const privateKey = new Uint8Array([
+  10, 20, 30, 40, 50, 60, 70, 80, 10, 20, 30, 40, 50, 60, 70, 80, 10, 20, 30,
+  40, 50, 60, 70, 80, 10, 20, 30, 40, 50, 60, 70, 80,
+]);
```

Comment:
Private keys are always secure and stored safely. For project purposes you can create a `.env` file to store it and refer here using `process.env.<VARIABLE_NAME>` 

Also make sure to never commit the `.env` file and have it in `.gitignore`